### ISSUE TITLE
Feature request: option to generate VOD from Live stream

### ISSUE THREAD

**Issue by maitreya2954** [google/shaka-packager#636]
Hi, 
I am trying to stream live and once the live stream ends to create a VOD manifest. I referred to this #321 and putting in required attributes works just fine. But it is quite cumbersome to parse the manifest file in java and get the values required to calculate mediaPresentationDuration, presentationTimeOffset. It gets worse with multiple representations. Its slowing down the pipeline.
So, It would be wonderful if an option is included to convert the manifest files(hls and mpeg-dash) to VOD after live stream or else just a separate option which takes in live stream manifest and generates VOD manifest.

**Comment by kqyang** [comment]
@maitreya2954 I'd like to understand more about your use case and how you want it to be supported. 

Would you like Shaka Packager to create a VOD manifest continuously or only once after the live stream ends?

**Comment by kqyang** [comment]
Mark as a duplicate of #321.
